lib/bigcommerce/storefront-config.ts

```diff
@@ -0,0 +1,29 @@
+import { BIGCOMMERCE_API_URL } from './constants';
+
+interface StorefrontTokenResponse {
+  data: {
+    token: string;
+  };
+  meta: unknown;
+}
+
+export const fetchStorefrontToken = async () => {
+  const response = await fetch(
+    `${BIGCOMMERCE_API_URL}/stores/${process.env.BIGCOMMERCE_STORE_HASH}/v3/storefront/api-token-customer-impersonation`,
```

Comment:
Also not critical, but we'll need to figure out a way how to not generate new token for every incoming request, which doesn't make sense. cc @zvuki

Reply:
I think we might be considering the approach of just generating a CIT and letting the merchant rotate it before (recommendation) or after it expires. That removes some complex logic with env vars we had in place before.